packages/ecs-agent/0005-bottlerocket-fix-procfs-path-on-host.patch

```diff
@@ -14,18 +14,18 @@ ECS agent in a container, the ECS agent can directly read the host's
  1 file changed, 1 insertion(+), 1 deletion(-)
 
 diff --git a/agent/ecscni/types_linux.go b/agent/ecscni/types_linux.go
-index 729acac..74ac96c 100644
+index 6d6ce353..056ede6a 100644
 --- a/agent/ecscni/types_linux.go
 +++ b/agent/ecscni/types_linux.go
-@@ -40,7 +40,7 @@ const (
- 	// ECSBranchENIPluginName is the binary of the branch-eni plugin
- 	ECSBranchENIPluginName = "vpc-branch-eni"
+@@ -47,7 +47,7 @@ const (
+ 	// ECSServiceConnectPluginName is the binary of the service connect plugin
+ 	ECSServiceConnectPluginName = "ecs-serviceconnect"
  	// NetnsFormat is used to construct the path to cotainer network namespace
 -	NetnsFormat = "/host/proc/%s/ns/net"
-+	NetnsFormat = "/proc/%s/ns/net"
++	NetnsFormat = "/host/%s/ns/net"
```

Comment:
Hmm, is this correct? I would expect it to still be under `/proc`, since the agent is running on the host and doesn't have any special mounts.

Reply:
Nice catch, I must have fumbled the edit here.